flink/v1.20/flink-runtime/NOTICE

```diff
@@ -63,29 +50,339 @@ NOTICE file:
 
 --------------------------------------------------------------------------------
 
-This binary artifact includes Apache Yetus with the following in its NOTICE
+This binary artifact includes Project Nessie with the following in its NOTICE
 file:
 
-| Apache Yetus
-| Copyright 2008-2020 The Apache Software Foundation
-|
-| This product includes software developed at
-| The Apache Software Foundation (https://www.apache.org/).
-|
-| ---
-| Additional licenses for the Apache Yetus Source/Website:
-| ---
-|
-|
-| See LICENSE for terms.
+| Nessie
+| Copyright 2015-2025 Dremio Corporation
+| 
+| ---------------------------------------
+| This project includes code from Apache Iceberg, with the following in its NOTICE file:
+| 
+| | Apache Iceberg
```

Comment:
Nit: Same as in the Spark PR, we can remove the Apache Iceberg portion of this notice to avoid recursion.

Reply:
Good catch, agree. Let me do that 😄

Reply:
I removed the Iceberg part from the Nessie `NOTICE` (but I keep Polaris and Netty sections).